ai-gateway/changelog.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Changelog
+icon: clock-rotate-left
+description: AI Gateway feature updates and changes
+---
+
+<Update label="2025-12-15" description="EA v0.0.1">
+  ## Early Access Release
+
+  Initial Early Access release of the AI Gateway.
+
+  ### Features
+
+  - OpenAI-compatible API for routing requests to multiple AI providers
+  - Support for OpenAI, Anthropic, Google, and other providers
+  - Multi-provider failover with automatic retries
```

Comment:
nit: technically we don't actually "retry" anything per-se since we don't have a retries config yet, but we do try different models/providers/api keys